src/dbwrapper.h

```diff
@@ -75,6 +75,8 @@ class CDBBatch
     friend class CDBWrapper;
 
 private:
+    static constexpr size_t kHeader{12}; // See: src/leveldb/db/write_batch.cc#L27
```

Comment:
This constant is unused now.

Reply:
Indeed, [removed](https://github.com/bitcoin/bitcoin/compare/aeda60c600d08e71e51fd251ee0435902caad8c4..e419b0e17f8acfe577c35c62a8a71a19aad249f3)